frontend/components/workflows/workflow-config-dialog.tsx

```diff
@@ -127,6 +129,32 @@ export function WorkflowConfigDialog({
             </form.Field>
           )}
 
+          {/* <div className="space-y-2">
```

Comment:
Could we just remove these commented implementations for now?

Reply:
done